plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -62,6 +67,35 @@ const ErrorNotice = () => {
 	);
 };
 
+const NoThanksButton = () => {
+	const [ isNoThanksClicked, setIsNoThanksClicked ] = useState( false );
+
+	return (
+		<Button
+			isBusy={ isNoThanksClicked }
+			isSecondary
+			onClick={ () => {
+				setIsNoThanksClicked( true );
+				recordEvent(
+					'wccore_subscriptions_empty_state_no_thanks_click',
+					{}
+				);
+				apiFetch( {
+					path: 'wc-admin/options',
```

Comment:
Looks like the class that handles these requests is deprecated - https://github.com/woocommerce/woocommerce/blob/add/experiment-subscriptions-admin-dismiss-button/plugins/woocommerce/src/Admin/API/Options.php#L15

Maybe we can use this method instead? https://github.com/woocommerce/woocommerce/blob/add/experiment-subscriptions-admin-dismiss-button/plugins/woocommerce-admin/client/payments-welcome/index.tsx#L251-L264

Reply:
nice catch! 

I've switched to use updateOptions like you mentioned from @woocommerce/data OPTIONS_STORE_NAME.